tox.ini

```diff
@@ -3,22 +3,18 @@
 # test suite on all supported python versions. To use it, "pip install tox"
 # and then run "tox" from this directory.
 
-[tox]
-# 3.5 is the lowest supported versions
-envlist = py35, py36, py37
```

Comment:
I need to look up what this actually changes, but it's past midnight. So I'll look this up tomorrow :)

Reply:
WoW! I assume that you are from the USA but looking at your profile and website, you are from the Netherlands.

Well, I was tentative to keep this without changes, but it is not really using because in the GitHub action `main.yml` / `build-linux.sh` you are specifying the version of Python `xvfb-run tox -v -e py"${PYTHON_VERSION//./}"` which is replaced by `xvfb-run tox -v -e py39`. Same case for macOS.

Furthermore, if you try to keep this `envlist` versions, they are not working because looks that you need to install via Tox or Ubuntu/macOS the different versions of Python. If you try to follow command `xvfb-run tox -v`, it will crash because it didn't find the Python versions. As, I showed in this screenshot:

![01-PythonVersions](https://user-images.githubusercontent.com/831380/198748703-acade103-2993-4daa-b24f-e0006ee36447.png)

Reply:
Makes sense! Thanks!